app/components/ChallengeDetailComponents/VisitorView.js

```diff
@@ -8,34 +8,70 @@ import {
 
 import MapView from 'react-native-maps';
 import accountCircle from '../../assets/accountcircle.png';
+import { Colors } from '../../theme/theme';
+import {ListDivider} from '../../components';
 
 export default class VisitorView extends Component {
   render() {
     return (
     <View>
-        <Text style = {{fontWeight: 'bold'}}>All Vistors</Text>
-
+        <ListDivider/>
+        <Text style = {{fontWeight: 'bold', fontSize: 15, paddingLeft:12,  backgroundColor: Colors.backgroundColor, paddingVertical:4, paddingBottom: 8}}>All Vistors</Text>
```

Comment:
Move this to a stylesheet. This is way to long to include right here. When reading it, I have to go all the way to the end of the line to figure out what is being displayed in this text box.

Another learning point here is that your lines shouldn't extend beyond 80-100 characters. There are linters that will prevent you from doing that, but generally speaking, if your code gets that long you should bump it down into a new line.